test/index.test.ts

```diff
@@ -0,0 +1,91 @@
+import 'isomorphic-fetch'
+import {
+    defaultChainOptions,
+    fetchBeacon,
+    fetchBeaconByTime,
+    HttpChainClient,
+    MultiBeaconNode,
+    watch
+} from '../lib'
+import fetchMock from 'jest-fetch-mock'
+import {testChain, validTestBeacon} from './data'
+import {StubChainClient} from './stub-chain-client'
+
+test('Beacon fetching works with testnet', async () => {
+    // connect to testnet
+    const node = new MultiBeaconNode('https://pl-us.testnet.drand.sh')
```

Comment:
Could we also get some tests against `mainnet` endpoints? Ideally both Cloudflare's and ours'.

Reply:
I can do some `fetchBeacon` tests, but not the `/chains` or `/info` as they aren't deployed on mainnet

Reply:
have added a copy of the testnet test but skipped, to be enabled when we release mainnet